backend/app/views/offer_mailer/email_nag.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+    </style>
+  </head>
+  <body>
+  <p>Dear <%= @first_name %> <%= @last_name %></p>
+  <p>
+    This is your <%= @nag_count %> reminder. Please accept 
```

Comment:
Does this come out as "This is your 1 offer" or "This is your 1st offer"?

Reply:
Good point!